spec/features/anonymous_user_happy_spec.rb

```diff
@@ -14,6 +14,18 @@
     expect(input).to have_content url_path(url)
   end
 
+  scenario 'that has already been made beautiful', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.